### DISCUSSION TITLE
Error in post-command hook when Corfu is used in LaTeX-buffer after a bracketed command

### DISCUSSION THREAD

**Discussion by nameiwillforget** [minad/corfu#466] [Q&A] [answered]
When writing LaTeX-documents, I frequently encounter the following error:
```
Error in post-command-hook (corfu--auto-post-command): (scan-error "Unbalanced parentheses" 26949 25790)
```
After this, Corfu-mode has to be toggled off and on again for it to provide completions. I'm not sure whether this is the only way to trigger it, but it happens reliably if I type after a bracketed command, then delete the text. For instance, if I write
```
\test{test}
```
then press Space after the closing bracket, I get that error. I tested this on the newest version of Corfu that is in Melpa.

**Comment by minad** [comment, marked as answer]
This is likely a bug in the latex completion function. You can try https://github.com/minad/corfu?tab=readme-ov-file#debugging-corfu to obtain a stack trace.
